docs/integration.md

```diff
@@ -0,0 +1,17 @@
+# JAG File Submission Continuous Integration
+
+## Git Action Workflow
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://app.lucidchart.com/documents/embeddedchart/cd317d74-4127-4518-8ad9-b99e7a66e005" id="M9w_SX3Zmq.f"></iframe></div>
+
+## Code Climate
+
+The JAG File Submission project uses [Code Climate](https://codeclimate.com) as its code quality tool. Much like SonarQube, Code Climate reads and interprets the code coverage reports generated by both the frontend and backend components of this project, and then returns a Pass/Fail status back to Github. This is an attempt to move the code quality gate in this projects development lifecycle to a point where the Maintainability and Test Coverage scores are as high as possible before this application reaches a production ready state.
```

Comment:
would be nice to say something about being run on pr before merge to master branch

Reply:
@sdevalapurkar-bcgov again